content/docs/reference/pkg/nodejs/pulumi/gcp/appengine/_index.md

```diff
@@ -60,26 +60,10 @@ Allows creation and management of an App Engine application.
    successfully deleted; this is a limitation of this provider, and will go away in the future.
    This provider is not able to delete App Engine applications.
 
-#### Example Usage
-
-```typescript
-import * as pulumi from "@pulumi/pulumi";
-import * as gcp from "@pulumi/gcp";
-
-const myProject = new gcp.organizations.Project("myProject", {
-    orgId: "1234567",
-    projectId: "your-project-id",
-});
-const app = new gcp.appengine.Application("app", {
-    locationId: "us-central",
-    project: myProject.projectId,
-});
-```
-
```

Comment:
Examples are now missing. @stack72

Reply:
we are losing examples due to them moving to TF0.12 HCL2 syntax
it's on the list to get fixed
it's going to continue happening